superset-frontend/src/components/FaveStar.tsx

```diff
@@ -31,6 +31,8 @@ interface FaveStarProps {
 
 const StyledLink = styled.a`
   font-size: ${({ theme }) => theme.typography.sizes.xl}px;
+  display: flex;
```

Comment:
Hmm... obviously this works, so I'm tempted to approve, but I have to ask out of curiosity:
1) Is there a reason to choose 'flex' over 'inline-block'?
2) Since you _are_ using flex, would `flex-directon: column` and `justify-content: space-around` negate the need to adjust the padding at all?